lib/main.coffee

```diff
@@ -9,6 +9,10 @@ module.exports =
         '.source.jade':
           default: false
       description: 'Automatically remove whitespace characters at ends of lines when the buffer is saved. To disable/enable for a certain language, use [syntax-scoped properties](https://github.com/atom/whitespace#readme) in your `config.cson`.'
+    removeTrailingWhitespaceAsYouType:
+      type: 'boolean'
+      default: false
```

Comment:
While I disagree that the setting should be enabled by default, I agree that it's more important to preserve previous defaults. I've committed the change.